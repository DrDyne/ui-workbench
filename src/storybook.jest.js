import styled from '@emotion/styled'
import initStoryshots, {
  multiSnapshotWithOptions
} from '@storybook/addon-storyshots'
import serializer from 'jest-emotion'

initStoryshots({
  suite: 'UI-WORKBENCH',
  test: multiSnapshotWithOptions(),
  snapshotSerializers: [serializer]
})

const MyButton = styled.button`
  color: royalblue;
  //color: orange; // breaks test
`

describe('jest-emotion serializer', () => {
  test('should render css', () => {
    const dom = render(<MyButton />)
    expect(dom).toMatchSnapshot()
  })
})
