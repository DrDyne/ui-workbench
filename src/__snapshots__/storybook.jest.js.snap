// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`jest-emotion serializer should render css 1`] = `
.emotion-0 {
  color: royalblue;
}

<button
  className="emotion-0 emotion-1"
/>
`;
